package.json

```diff
@@ -674,6 +677,7 @@
     "process": "^0.11.10",
     "pumpify": "^2.0.1",
     "randomcolor": "^0.5.4",
+    "react-compiler-webpack": "^0.2.0",
```

Comment:
pin this to exactly `0.2.0`. In semver versioning the `0` version allows for breaking changes at any time.